cmd/root.go

```diff
@@ -125,6 +125,7 @@ func init() {
 	RootCmd.PersistentFlags().StringVarP(&utils.Flags.Key, "key", "k", "", wski18n.T(wski18n.ID_CMD_FLAG_KEY_FILE))
 	RootCmd.PersistentFlags().StringVarP(&utils.Flags.Cert, "cert", "c", "", wski18n.T(wski18n.ID_CMD_FLAG_CERT_FILE))
 	RootCmd.PersistentFlags().BoolVarP(&utils.Flags.Managed, "managed", "", false, wski18n.T(wski18n.ID_CMD_FLAG_MANAGED))
+	RootCmd.PersistentFlags().StringVarP(&utils.Flags.Name, "name", "", "", wski18n.T(wski18n.ID_CMD_FLAG_NAME))
```

Comment:
Understand why the functionality is needed, but the flag "name" seems too generic IMO.  I would prefer "project" at min. or better "projectname" (fully self-explanatory).